components/events.jsx

```diff
@@ -102,8 +112,17 @@ export default () => {
     fetchData();
   }, [selectedOption, nextPageToken, prevPageToken]);
 
+  useEffect(() => {
+    fetchUser();
+  }, [events])
+
   return (
     <div className="container" style={{ marginTop: '50px' }}>
+      <div style={{textAlign: 'center'}}>
+      <Link to={`/user_busy_times?user=${events?.length && events[0].event_memberships[0].user}`}>
```

Comment:
Should this be using the user URI from `fetchUser`?

Reply:
Ah, good idea! I incorporated that GET User endpoint last week, and this line is at least a couple of months old, I think. Haha. I'll change it! Thank you.